src/content/drafts/eslint-community-org.md

```diff
@@ -0,0 +1,131 @@
+originally authored by [@mysticatea](https://github.com/mysticatea) as well:
+
+* [`eslint-plugin-es`](https://github.com/mysticatea/eslint-plugin-es)  
+  We'll work on getting all
+  [`eslint-plugin-es-x`](https://github.com/eslint-community/eslint-plugin-es-x)
+  features merged as well
+* [`eslint-plugin-eslint-comments`](https://github.com/mysticatea/eslint-plugin-eslint-comments)
+* [`eslint-plugin-node`](https://github.com/mysticatea/eslint-plugin-node)  
+  We'll work on getting all
+  [`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n)
+  features merged as well
+
+Of course, we don't want to stop there.
+
+If you think your project would be a good fit for the new community org, *or*
+you're depending on a project that's currently unmaintained and want to have the
+community to take it over, you can contact the Community Core team on
+[the official ESLint Discord](https://eslint.org/chat).
+
+Keep in mind that we'll only be able to take over projects that are adhering to
+the following acceptance criteria:
+
+* Is it a package that is ESLint-related?  
+  Mostly this will be ESLint plugins, but (unmaintained) dependencies of such
+  packages, closely related packages or packages split from the main ESLint repo
+  (like
+  [`eslint-formatter-codeframe`](https://github.com/eslint-community/eslint-formatter-codeframe)
+  or
+  [`eslint-formatter-table`](https://github.com/eslint-community/eslint-formatter-table))
+  or used by the main repo (like
+  [`eslint-utils`](https://github.com/mysticatea/eslint-utils) and
+  [`regexpp`](https://github.com/mysticatea/regexpp)) could go in the
+  `@eslint-community` GitHub organization as well.
+
+  Because plugins can expose their own (recommended) configs, we already accept
+  config packages in some way.  
+  However, the community core team won't be eager to accept shared ESLint config
+  packages, as they can be very opinionated.
+
+  It also doesn't matter if the package is purely TypeScript-related or not, as
+  we don't want to fragment the ESLint community more without any real benefit.
+
+* Is it widely depended upon throughout the ESLint community?  
+  We don't have a real number in mind here, but the packages we currently took
+  over (or that we're planning to take over) almost all have at least 3M
+  downloads/week.
+
+* Is it currently unmaintained?  
+  We don't want to take over packages that are still maintained by their
+  original author(s), as that would be a waste of time for everyone involved.
+
+These criteria aren't written in stone, but they're good guidelines to see if we
+should take over a package or not.
+
+If we decide to take over a package, but we can't get into contact with the
+owner of the repo to transfer ownership over to the new GitHub organization, we
+will (temporarily) fork the repo and publish it under the `@eslint-community` npm
+organization. However, the ultimate goal will always be to transfer the original
+repo.
```

Comment:
This reads as very aggressive, and I think we should just eliminate it altogether.
```suggestion
```